client/src/components/SocialMediaShow.js

```diff
@@ -61,6 +61,57 @@ const SocialMediaShow = (props) => {
     }
   }
 
+  const handleCommentDelete = async (commentId) => {
+    try { 
+      const response = await fetch(`/api/v1/websites/${socialMediaShow.id}/comments/${commentId}`, { method: "DELETE" })  
```

Comment:
we should be able to shorten this to `/api/v1//comments/${commentId}` on a regular comments router.

That said, ill allow it, but this would be better with the shorter path, as we only need the id of the comment to delete it